src/SFA.DAS.ProviderRegistrations/Services/EmployerApprenticeshipApiHttpClientFactory.cs

```diff
@@ -1,20 +1,20 @@
-﻿using SFA.DAS.Http;
-using SFA.DAS.ProviderRegistrations.Configuration;
-
-namespace SFA.DAS.ProviderRegistrations.Services
-{
-    class EmployerUsersApiHttpClientFactory : IEmployerUsersApiHttpClientFactory
-    {
-        private readonly EmployerUsersApiClientSettings _employerUsersApiClientSettings;
-
-        public EmployerUsersApiHttpClientFactory(EmployerUsersApiClientSettings employerUsersApiClientSettings)
-        {
-            _employerUsersApiClientSettings = employerUsersApiClientSettings;
-        }
-
-        public IRestHttpClient CreateRestHttpClient()
-        {
-            return new RestHttpClient(new AzureActiveDirectoryHttpClientFactory(_employerUsersApiClientSettings).CreateHttpClient());
-        }
-    }
-}
+﻿using SFA.DAS.Http;
+using SFA.DAS.ProviderRegistrations.Configuration;
+
+namespace SFA.DAS.ProviderRegistrations.Services
+{
+    class EmployerApprenticeshipApiHttpClientFactory : IEmployerUsersApiHttpClientFactory
+    {
+        private readonly EmployerApprenticeshipApiClientSettings _employerUsersApiClientSettings;
```

Comment:
is this variable name right with the change in class name?

Reply:
Fixed